tests/integration/block_createBlock_k8s_test.go

```diff
@@ -172,8 +176,130 @@ func (s *K8sBlockImageCreateSuite) TestCreateSamePVCTwice() {
 
 }
 
-func (s *K8sBlockImageCreateSuite) tearDownTest(claimName string, poolName string, storageClassName string) {
-	installer.BlockResourceOperation(s.kh, installer.GetBlockPvcDef(claimName, storageClassName), "delete")
+// Test case when persistentvolumeclaim with ReadOnlyMany
+func (s *K8sBlockImageCreateSuite) TestCreateReadOnlyManyPVCWhenStorageClassExists() {
```

Comment:
this test looks duplicated with the next test, except it is RO instead of RW. Can you factor this into a helper method to avoid the duplication?